test/unit/rules-elements.html

```diff
@@ -0,0 +1,192 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/validator/4.0.5/validator.js"></script>
```

Comment:
Since this was more of a proposal then a serious PR. I was shooting to make the test more like code samples then test. Realistically, If I where writing a test for this, I wouldn't need to use a lib as a normal behavior would have been fine.. However, the goal was to demo how we could utilize existing validation frameworks to beef up polymer.